injected/src/features/web-telemetry.js

```diff
@@ -15,6 +18,10 @@ export class WebTelemetry extends ContentFeature {
         }
     }
 
+    urlChanged(navigationType = 'unknown') {
+        this.fireTelemetryForUrlChanged(navigationType);
```

Comment:
Should we wrap this in config to disable it remotely if needed?

```suggestion
        if (this.getFeatureSettingEnabled('urlChanged')) {
            this.fireTelemetryForUrlChanged(navigationType);
        }
```

Happy if you think the answer is no btw :)

Reply:
My bad, yeah I think it should be